src/chain.h

```diff
@@ -144,6 +146,15 @@ class CBlockIndex
     //! (memory only) The anchor for the tree state up to the end of this block
     uint256 hashAnchorEnd;
 
+    //! Change in value held by the Sprout circuit over this block.
+    //! Will be boost::none for older blocks on old nodes until a reindex has taken place.
```

Comment:
So we're changing the blocks retroactively and adding this value to all of them?

Reply:
We are changing the block indices retroactively, not the blocks themselves. For older indices, if/when they are rewritten this value will be `boost::none` until a full reindex is done.